src/gam/gapi/cloudidentity/inboundsso.py

```diff
@@ -251,24 +252,19 @@ def create_credentials():
         elif myarg == 'keysize':
             key_size = int(sys.argv[i+1])
             if key_size not in allowed_sizes:
-                controlflow.expected_argument_exit('key_size',
-                                                   allowed_sizes,
-                                                   key_size)
+                controlflow.expected_argument_exit('key_size', allowed_sizes, key_size)
```

Comment:
wrap should stay in place unless we can keep a single line under 80 char

Reply:
Do you want me to unwrap the over 80 char ones?